actionlist/app/reducers/action_items.js

```diff
@@ -1,13 +1,93 @@
 /* eslint arrow-body-style: ["error", "as-needed", { "requireReturnForObjectLiteral": true }] */
 import { handleActions } from 'redux-actions';
-import { TOGGLE_DRAWER } from '../actions/types';
+import { TOGGLE_DRAWER, SORT_BY_CREATION_DATE, SORT_BY_LAST_APPROVED_DATE, SORT_BY_PROCESS_TYPE, SORT_BY_ACTION_REQUIRED } from '../actions/types';
 import { processInstances } from '../lib/commons';
 
 export const defaultState = {
   dataSource: processInstances,
   drawerExpanded: false,
+  sortValue: '',
 };
 
+const sortByCreationDate = (state) => {
```

Comment:
Sure thing. Since sorting just alters the order of the data we have done it this way. Would take care of it while implementing filters.